tests/system/datastore/test_v3io.py

```diff
@@ -185,8 +201,21 @@ def test_v3io_large_object_put(self):
             f"test_v3io_large_object_put: total time: {time.time() - first_start_time} seconds"
         )
 
-    def test_put_get_and_download(self):
-        data_item, _ = self._get_data_item()
+    @pytest.mark.parametrize("use_secrets_as_parameters", [True, False])
+    def test_put_get_and_download(
+        self, use_datastore_profile, use_secrets_as_parameters
+    ):
+        secrets = {}
+        if use_secrets_as_parameters:
+            os.environ["V3IO_ACCESS_KEY"] = "wrong_token"
+            # Verify that we are using the profile secret:
```

Comment:
Maybe clarify that we set a wrong token to make sure that the right token is picked from the datastore profile.